sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/randomExpressions.scala

```diff
@@ -181,3 +187,239 @@ case class Randn(child: Expression, hideSeed: Boolean = false) extends RDG {
 object Randn {
   def apply(seed: Long): Randn = Randn(Literal(seed, LongType))
 }
+
+@ExpressionDescription(
+  usage = """
+    _FUNC_(min, max[, seed]) - Returns a random value with independent and identically
+      distributed (i.i.d.) values with the specified range of numbers. The random seed is optional.
+      The provided numbers specifying the minimum and maximum values of the range must be constant.
+      If both of these numbers are integers, then the result will also be an integer. Otherwise if
+      one or both of these are floating-point numbers, then the result will also be a floating-point
+      number.
+  """,
+  examples = """
+    Examples:
+      > SELECT _FUNC_(10, 20) > 0 AS result;
+      true
+  """,
+  since = "4.0.0",
+  group = "math_funcs")
+object UniformExpressionBuilder extends ExpressionBuilder {
```

Comment:
Why do you need an expression builder? Can't you just define an expression w/ a few constructors? Please, explain this.

Reply:
I originally tried that, but found that putting the random seed expression as a case class field made it get printed in all sorts of places (even if I overrode `sql` and/or `toString`). The value is non-deterministic which would cause flakiness. So I moved the seed to a private field of the class instead, but this precludes setting it in extra constructors, thus arriving at this approach.

Reply:
@MaxGekk update: alright I was able to remove both the `ExpressionBuilder` and the `var` and just add the random seed as a class field, per your suggestion.